implementations/rust/ockam/ockam_api/src/nodes/models/services.rs

```diff
@@ -63,3 +63,43 @@ impl<'a> StartAuthenticatedServiceRequest<'a> {
         }
     }
 }
+
+/// Request body when instructing a node to start an Uppercase service
+#[derive(Debug, Clone, Decode, Encode)]
+#[rustfmt::skip]
+#[cbor(map)]
+pub struct StartUppercaseServiceRequest<'a> {
+    #[cfg(feature = "tag")]
+    #[n(0)] tag: TypeTag<5179596>,
```

Comment:
These numbers should be unique. You can use this page to generate a valid TypeTag: https://www.random.org/integers/?num=100&min=1000000&max=9999999&col=5&base=10&format=html&rnd=new

Reply:
fix in afb3e03